src/snapred/ui/workflow/WorkflowImplementer.py

```diff
@@ -13,6 +15,8 @@
 
 
 class WorkflowImplementer:
+    _dryRun = Config["ui.workflow.dryRun"]
```

Comment:
Is this (including the stubbed request below) OK to include here? This looks like it's just used for testing

Reply:
I could probably remove this stuff, I was tinkering with the idea of being able to set a development var to dryrun the ui to make ui/ux testing smoother, but this implementation is incomplete.